packages/create-bison-app/template/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  postgres:
+    container_name: postgres
+    ports:
+      - 5432:5432
```

Comment:
I'm wondering if we should change the port. It could be the user is running a local version of Postgres. This would fail.